<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Company</class>
 <widget class="QDialog" name="Company">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_22">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Название</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Полное</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLineEdit" name="companyTitle"/>
             </item>
             <item>
              <widget class="QLineEdit" name="companyFullTitle"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ИНН</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="INN"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>КПП</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="KPP"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Юр.Адрес</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Факт.адрес</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Грузо-тель</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLineEdit" name="companyUrAddress"/>
             </item>
             <item>
              <widget class="QLineEdit" name="companyFactAddress"/>
             </item>
             <item>
              <widget class="QLineEdit" name="companyGruz"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Телефоны</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Факс</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLineEdit" name="companyPhones"/>
               </item>
               <item>
                <widget class="QLineEdit" name="companyFax"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Метро</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>E-Mail</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLineEdit" name="companyMetro"/>
               </item>
               <item>
                <widget class="QLineEdit" name="companyMail"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Расч.счет</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Банк</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLineEdit" name="companyRasSchet"/>
             </item>
             <item>
              <widget class="QLineEdit" name="companyBank"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>Кор.счет</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Директор</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Гл.бух.</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <widget class="QLineEdit" name="companyKorSchet"/>
               </item>
               <item>
                <widget class="QLineEdit" name="companyDirector"/>
               </item>
               <item>
                <widget class="QLineEdit" name="companyBuh"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>БИК</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>ОКВЭД</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>ОКПО</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QLineEdit" name="BIK"/>
               </item>
               <item>
                <widget class="QLineEdit" name="OKVED"/>
               </item>
               <item>
                <widget class="QLineEdit" name="OKPO"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>ФИО ПБОЮЛ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Св-во ПБОЮЛ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Контакт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Дополнение</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <item>
            <widget class="QLineEdit" name="FIOPBOUL"/>
           </item>
           <item>
            <widget class="QLineEdit" name="properPBOUL"/>
           </item>
           <item>
            <widget class="QLineEdit" name="contact"/>
           </item>
           <item>
            <widget class="QLineEdit" name="DLC"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>194</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="companyOK">
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="companyChange">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="companyCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>193</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
